.github/scripts/cherry-pick-helper.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# .github/scripts/cherry-pick-helper.sh
+# Utility script for manual cherry-pick operations
+
+set -euo pipefail
+
+COMMIT_SHA="$1"
+TARGET_BRANCH="$2"
+SHORT_SHA="${COMMIT_SHA:0:7}"
+BRANCH_NAME="cherry-pick/${SHORT_SHA}-to-${TARGET_BRANCH}"
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+log_info() { echo -e "${GREEN}ℹ️  $1${NC}"; }
+log_warn() { echo -e "${YELLOW}⚠️  $1${NC}"; }
+log_error() { echo -e "${RED}❌ $1${NC}"; }
```

Comment:
```suggestion
log_info() { echo -e "${GREEN}ℹ️  $1${NC}"; }
log_warn() { echo -e "::warning:: $1"; }
log_error() { echo -e "::error:: $1"; }
```

https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#example-creating-an-annotation-for-an-error

Reply:
this script is for manual action and not run in CI. as such, i don't think the GHA specific syntax `::warning::` is warranted. wdyt?

Reply:
Agreed if that's the case. Wondering though if we can use it for local + CI?